vignettes/correlation.Rmd

```diff
@@ -16,36 +16,48 @@ state-level COVID-19 case and death rates, smoothed using 7-day trailing
 averages.
 
 ```{r, message = FALSE, warning = FALSE}
-library(epidatr)
 library(epiprocess)
 library(dplyr)
 ```
 
-The data is fetched with the following query:
-```{r, message = FALSE}
+The data is included in this package (via the [`epidatasets` package](https://cmu-delphi.github.io/epidatasets/)) and can be loaded with:
+```{r}
+x <- cases_deaths_subset %>%
```

Comment:
question: This isn't the same data; this vignette apparently used all states, not just 6.  Was this change intentional?  (& if so, have you already checked sanity-checked new results and any statements about dimensions of the input?)

Reply:
Not intentional, thanks for pointing thisout